spectro-application-profile.tf

```diff
@@ -0,0 +1,70 @@
+locals {
+  application_profiles = toset(flatten([
```

Comment:
please put comment of what is mapped to what in this variable and source (i.e. application-profile.yaml)

Reply:
addressed